pandajedi/jediorder/JediMsgProcessor.py

```diff
@@ -24,5 +24,5 @@ def launcher():
         tmp_log.error('failed to read config json file; should not happen... {0}: {1}'.format(e.__class__.__name__, e))
         raise e
     else:
-        agent = MsgProcAgent(config_file)
+        agent = MsgProcAgent(config_file, process_sleep_time=0.0001)
```

Comment:
Just to make sure. Is there any problem if poocess_sleep_time is set to 0?
Will it be too CPU consuming?

Reply:
Just tried process_sleep_time=0 on a node; then the msgprocessor process overuses a core (CPU > 100%)

```
  PID USER      PR  NI    VIRT    RES    SHR S  %CPU %MEM     TIME+ COMMAND                                                                             
14871 atlpan    20   0 1923020 140424  19148 S 123.3  0.9   3:09.36 python                                 
```

while with process_sleep_time=0.0001 , msgprocessor uses CPU 20 ~ 30%.

Overusing a core does not seem to influence other agents (system load is fine), but looks unnecessary to me.

Reply:
OK, thanks for checking. Then isn't it better to set the default value to

https://github.com/PanDAWMS/panda-common/blob/master/pandacommon/pandamsgbkr/msg_processor.py#L61

like  def __init__(self, attr_dict, sleep_time=0.0001):

to avoid specifying 0.0001 to every instance?

Reply:
OK, as long as 0.0001 is reasonable for most use cases asides from JEDI's.

Also updated panda-common

https://github.com/PanDAWMS/panda-common/pull/18

Reply:
Looks good now.